scripts/mkcloud

```diff
@@ -1308,6 +1309,9 @@ function expand_steps
                     _testupdate|testupdate)
                         runcmds="$runcmds addupdaterepo runupdate testsetup cct"
                     ;;
+                    _test_setuphost)
+                        runcmds="$runcmds cleanup prepare setupadmin onadmin+test_setuphost"
```

Comment:
nay, this is calling the "onadmin" function with parameter "test_setuphost" which in the end will run the onadmin_test_setuphost function in the admin VM